README.md

```diff
@@ -109,6 +109,60 @@ sharing that same array or object. Uncovering this issue is not always an easy
 task and so *getDefaultProps* was also implemented (thanks to the React team for
   this concept as well).
 
+
+## Test Helper
+
+This addon provides a `validatePropTypes` test helper that can be used in unit
+tests to validate that the `propTypes` definitions on your Components are as
+expected.  It will check for definitions that are different, missing, and
+unexpected.
+
+The first argument can either be a Component class or instance, with the latter
+being for those times that you need to set properties to satisfy instantiation
+guard clauses or other.
+
+The second argument is your expected `propTypes` definition.
+
+It's important to note the setup of the `describe()` in relation to the
+`validatePropTypes` call.  The call is not wrapped in an `it()` nor does it
+call `expect()`, as the helpers does so itself.
```

Comment:
The sentence "The call is not wrapped in an `it()` nor does it call `expect()`, as the helpers does so itself." is a little confusing. I'm assuming what you mean is that the consumer of `validatePropTypes` doesn't have to wrap it in an `it()` or call `expect()` because `validatePropTypes` will have it's own `it()` and `expect()` but that's not really how it reads, it reads as contradictory, since it says first that "the call" which I believe refers to the call to `validatePropTypes()` doesn't call `expect()` and then it says "the helpers does so itself" which sounds like `validatePropTypes()` *does* call `expect()`